### PR TITLE
fix: cypress verify failure

### PR DESCRIPTION
**Pull request by ItsANameToo**
~Applies a workaround to hopefully have the cypress tests run again, based on https://github.com/cypress-io/cypress/issues/6184#issuecomment-577308973~

They just released their new version, so updating to that will fix the issue

### CODE REVIEW

**Comment by leMaur** [comment]
@alfonsobries It sounds good, even compared to Figma! 👍🏽

**Comment by alexbarnsley** [comment]
@alfonsobries I'm going to mark as draft until the the UI dep is updated. I'd have updated it but I have a PR that also updates that dependency so figured I'd just wait

**Comment by alfonsobries** [comment]
Just updated the ui dependency should be ready now

File: resources/views/components/general/search/search.blade.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
     @search-type-changed.window="searchType = $event.detail"
     class="w-full"
 >
-    <div class="flex flex-col bg-white rounded-xl dark:bg-theme-secondary-900">
+    <div class="flex flex-col bg-white rounded-lg md:shadow-lg-smooth dark:bg-theme-secondary-900">
```
Comment (ItsANameToo) [inline resources/views/components/general/search/search.blade.php]: shouldn't this be rounded-xl? afaik we don't use rounded-lg much/at all

Comment (alfonsobries) [reply resources/views/components/general/search/search.blade.php]: @ItsANameToo the design uses less roundiness: 

### Current implementation (with xl)
image

### With `lg` roundiness

image


### Design
image

Comment (alfonsobries) [reply resources/views/components/general/search/search.blade.php]: @ItsANameToo so according to Oleg lg is correct

Comment (ItsANameToo) [reply resources/views/components/general/search/search.blade.php]: @alfonsobries according to oleg it should be 12px, so `xl` :trollface: I have updated it on your PR

**Pull request merged**